packages/terra-aggregator/src/terra-dev-site/doc/example/CustomRenderAggregatorExample.jsx

```diff
@@ -27,18 +27,18 @@ const CustomRenderAggregatorExample = () => (
       render={renderData => (
         <Grid>
           <Grid.Row>
-            <Grid.Column col={6}>
+            <Grid.Column tiny={6}>
```

Comment:
tiny={6}? What is going on here?

Reply:
The `col` prop changed to `tiny` in terra-grid v6. https://github.com/cerner/terra-core/blob/master/packages/terra-grid/docs/UPGRADEGUIDE.md#changed